en/python_installation/instructions.md

```diff
@@ -108,11 +108,11 @@ Verify the installation was successful by opening a command prompt and running t
 {% filename %}command-line{% endfilename %}
 ```
 $ python3 --version
-Python 3.6.1
+Python {{ book.py_release }}
 ```
-The version shown may be different from 3.6.1 -- it should match the version you installed.
+The version shown may be different from {{ book.py_release }} -- it should match the version you installed.
 
-**NOTE:** If you're on Windows and you get an error message that `python3` wasn't found, try using `python` (without the `3`) and check if it still might be a version of Python that is 3.4.0 or higher. If that doesn't work either, you may open a new command prompt and try again; this happens if you use a command prompt left open from before the Python installation. 
+**NOTE:** If you're on Windows and you get an error message that `python3` wasn't found, try using `python` (without the `3`) and check if it still might be a version of Python that is 3.4.0 or higher. If that doesn't work either, you may open a new command prompt and try again; this happens if you use a command prompt left open from before the Python installation.
```

Comment:
[Django 3.1 requires Python 3.6 or greater](https://docs.djangoproject.com/en/3.1/faq/install/#faq-python-version-support).
```suggestion
**NOTE:** If you're on Windows and you get an error message that `python3` wasn't found, try using `python` (without the `3`) and check if it still might be a version of Python that is 3.6.0 or higher. If that doesn't work either, you may open a new command prompt and try again; this happens if you use a command prompt left open from before the Python installation.
```

(Maybe we should introduce a variable for the minimum Python version, too?)

Reply:
@das-g Check commit https://github.com/DjangoGirls/tutorial/commit/d90b4a38a4531b28701c866cff611f1cc7c8161e